web.test.tests/web.test.tests/Tests/DepositePageTests.cs

```diff
@@ -266,5 +254,37 @@ public void Field_Captions_th(String caption)
             //Assert
             Assert.True(driver.FindElement(By.XPath($"//table//tr/th[text() = '{caption}']")).Displayed);
         }
+
+
+        [TestCase(0, 0, 0)]
+        [TestCase(0, 0, 1)]
+        [TestCase(0, 1, 0)]
+        [TestCase(0, 1, 1)]
+        [TestCase(1, 0, 0)]
+        [TestCase(1, 0, 1)]
+        [TestCase(1, 1, 0)]
+        [TestCase(100001, 1, 1)]
+        [TestCase(1, 101, 1)]
+        [TestCase(1, 1, 366)]
+        public void CalculateBtnDisabled(int amount, int rate, int term)
+        {
+            DepositPage depositPage = new DepositPage(driver);
+            depositPage.SetAmount(amount);
+            depositPage.SetRate(rate);
+            depositPage.SetTerm(term);
+
+            Assert.IsFalse(depositPage.CalculateBtn.Enabled);
+        }
+
+        [TestCase(1, 1, 1)]
+        public void CalculateBtnEnabled(int amount, int rate, int term)
```

Comment:
You don't need this test. If the 'Calculate' button is disabled, the 'Income_And_Interest' test will fail.

Reply:
CalculateBtnEnabled test is created in order to simplify 'Income_And_Interest' test.
 Cases with invalid input data (i.e. when 'Calculate' button is disabled) are not considered in 'Income_And_Interest' test.